chromadb/api/types.py

```diff
@@ -53,16 +54,17 @@ def maybe_cast_one_to_many_ids(target: OneOrMany[ID]) -> IDs:
 
 
 # Embeddings
+PyEmbedding = PyVector
 Embedding = Vector
```

Comment:
may be out of scope but the proliferation of types and indirection here is kinda confusing, wondering if just having `PyVector` and `NpVector` instead would work

Reply:
I think out of scope for this PR—it would require changing quite a bit more code across the board and introduce more risk. I also think there's a good reason to keep the idea of an `Embedding` separate from that of a `Vector`. While the former is always the latter, there are usages of `Vector` across the code base where it is not an `Embedding`.

Reply:
Yeah agree this is weird but I guess the distinction being made is "vector" is a array of numbers vs "embedding" is a domain specific input to chroma that may change. Which makes sense to separate.